app/components/Home.js

```diff
@@ -1,6 +1,16 @@
-import React from 'react'
-export default () => (
-  <div>
-    <h1>Deer</h1>
-  </div>
-)
+import React, { Component } from 'react'
+import electron from 'electron'
+
+export default class Home extends Component {
+  render () {
+    // when the user hits home page we do not want
+    // to redisplay welcome page again
+    let electronStore = electron.remote.getGlobal('sharedObj').electronStore
```

Comment:
Currently, electron store works on both main and renderer process. You just need to import and use it directly